packages/client/hmi-client/src/components/model/petrinet/tera-parameter-entry.vue

```diff
@@ -124,23 +112,29 @@ function getSourceLabel(initialId) {
 }
 
 function formatPayloadFromParameterChange(parameters) {
-	const distribution = getParameterDistribution(props.modelConfiguration, props.parameterId);
+	const parameterDistribution = getParameterDistribution(
```

Comment:
Is there a need to specify that its a `parameterDistribution` if we already in the `tera-parameter-entry` component?

Reply:
I just needed a new name cause I created that `distribution` computed value and the linter was whining. I figured `parameterDistribution` would be okay instead

Reply:
I could clone it 😄